index.html

```diff
@@ -163,53 +163,62 @@ <h2 class='tagline'>Sample Pages</h2>
               path: 'inline-session.html',
               description: 'Demonstrates use of an "inline" XRSession to present the same WebGL scene on the page and on an XR device.' },
 
-            { tag: 'hr' },
-            { tag: 'br' },
-
-            { title: 'Reduced-Bind Rendering', category: 'Performance',
-              path: 'reduced-bind-rendering.html',
-              description: 'Demonstrates a technique to reduce the number of state changes made while rendering.' },
-
```

Comment:
Please don't reorder the samples on this page. (For the rest of the pages it matters less.)

If you are reading through the sample code there were certain concepts introduced in this sample, for example, that were then built on by the ones that followed it. As the samples get more into demonstrating specific types of content or modules it begins to matter less, but the dozen or so of these had a natural order to them that I'd prefer not be changed.

Reply:
Gotcha, will put those back then